acceptance-tests/src/test/resources/features/site/homepage.feature

```diff
@@ -0,0 +1,9 @@
+@homepage
```

Comment:
Is the tag still needed or left behind by accident?

Reply:
Removed. Thanks!